docs/mining/proof-of-stake.md

```diff
@@ -71,65 +71,34 @@ up for a pool.
-
-<i class="fa fa-exclamation-triangle"></i> ** It is highly recommended
-to try dcrticketbuyer on testnet to make sure it's correctly configured and
-behaves as expected before attempting to use it on mainnet. Once tickets are
-purchased, the funds can be locked for months without any way to retrieve
-them. **
-
-> dcrticketbuyer (Solo Mining)
-
-The bare minimum configuration is to provide the RPC access credentials for
-dcrd and dcrwallet.  In this configuration, dcrticketbuyer will use all
-available funds to purchase tickets at an average price that is calculated
-by examining historic prices.
-
-```no-highlight
-dcrduser=<dcrd RPC Username>
-dcrdpass=<dcrd RPC Password>
-dcrwuser=<dcrwallet RPC Username>
-dcrwpass=<dcrwallet RPC Password>
-```
-
-> dcrticketbuyer (Pool Mining)
-
-Purchasing tickets and delegating the voting rights to a stake pool requires
-the following stake pool-specific configuration options.  The values to use
-may be found in the instructions portion of the Tickets page that is present
-on any of the official stake pools.
-
-```no-highlight
-pooladdress=DsExamplePaymentAddr
-poolfees=7.5
-ticketaddress=DcExampleTicketAddr
-```
-
-> dcrwallet
-
-<i class="fa fa-thumbs-down"></i> **This method of using dcrwallet to
-purchase tickets automatically has been deprecated and is not recommended.**
-
-<i class="fa fa-exclamation-triangle"></i> **Automatic ticket purchasing built-in to dcrwallet can incur transaction fees beyond your control!**
-
-Automatic ticket purchasing can be enabled by adding the relevant configuration options to dcrwallet's configuration file.
-See [dcrwallet.conf configuration](/advanced/storing-login-details.md#dcrwallet).
-
-```no-highlight
-; Always keep this amount of DCR in the wallet.
-balancetomaintain=10
-; Enable ticket purchasing and voting.
-enablestakemining=1
-; Prompt for a password on startup to ensure the wallet is unlocked immediately.
-promptpass=1
-; Maximum amount to spend on a ticket.
-ticketmaxprice=10
-```
+You can set up dcrwallet to automatically purchase tickets on your behalf. We recommended you read
+and understand the options available before using the feature as you may set your fees and ticket 
+prices higher than desired.
+
+All of these options can be specified on the command line or in dcrwallet.conf. Note that at
+this time there is no way to change settings from within dcrwallet. You will need to restart it to 
+adjust your settings.
+
+Parameter|Description|Default|Explanation
+:----------:|:---------------------------:|:----------:|:---------------------------:
+ticketbuyer.maxpricescale|Attempt to prevent the stake difficulty from going above this multiplier (>1.0) by manipulation, 0 to disable|2|If purchasing tickets at this price window would cause the next window to be higher than ```(price * multiplier)```, do not buy tickets. This may cause you not buy tickets when otherwise you could. Recommend to set to 0.
```

Comment:
This and pricetarget have recently been deemed DEPRECATED. I believe your comments here still apply, but adding a note that they are currently depracated and will be removed in a future release